packages/dev-server-storybook/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @web/dev-server-storybook
 
+## 0.3.8
+
+### Patch Changes
```

Comment:
@Westbrook should we change this to `0.4.0` to indicate the version bump contains breaking changes?

Reply:
What breaking change?

The point of the approach I took to this update was that it evenly support both sides of the `lit@2.0` release, and I've shown in my work as well as working with some other people in their projects that that's true. However, I could have missed some new news here...

I agree this is a "feature" but pre-1.0 you don't really have ability to show that very well.

Reply:
Ok, thanks. I thought bump for storybook-prebuilt alpha is a breaking change. But during 0.x it's probably fine.